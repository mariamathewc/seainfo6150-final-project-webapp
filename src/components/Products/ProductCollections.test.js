import React from "react";

import { render } from "@testing-library/react";
import routeData from 'react-router';

const mockLocation = {
	pathname: 'localhost:3000/collections',

}


describe("Product Collection test", () => {
	it("renders correctly", () => {
		const { container } = jest.spyOn(routeData, 'useLocation').mockReturnValue(mockLocation);
		expect(container).toMatchSnapshot();
	});
});

