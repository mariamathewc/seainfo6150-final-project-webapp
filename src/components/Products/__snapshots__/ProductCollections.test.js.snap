// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Collection tests renders correctly 1`] = `undefined`;
